gulpfile.js

```diff
@@ -42,4 +44,17 @@ gulp.task('serve', function() {
     }));
 });
 
+gulp.task('sass', function() {
+  gulp.src(['./sass/*.sass', './sass/*.scss'])
+      .on('error', function(e) {
+        handleError(e, this);
+      })
+      .pipe(sourcemaps.init())
+      .pipe(sass())
+      .pipe(sass({indentationSyntax: true})
+      .pipe(autoprefixer('last 2 versions'))
+      .pipe(sourcemaps.write('./'))
```

Comment:
Is this putting the sourcemaps into a different directory than the css output? I would think they would go in the same directory, am I missing something?